searchcore/src/vespa/searchcore/config/ranking-expressions.def

```diff
@@ -0,0 +1,5 @@
+# Copyright 2017 Yahoo Holdings. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+namespace=vespa.config.search.core
+
```

Comment:
I wonder if it should be put in configdefinitions together with rank-profiles.def. I guess it might be need in java too.

Reply:
I put it where the ranking constants already are located

Reply:
I noticed afterwards. Agree Better keep them close as they are the same.